bazel-mode.el

```diff
@@ -73,6 +73,20 @@
   (when bazel-mode-buildifier-before-save
     (bazel-mode-buildifier)))
 
+(defconst bazel-mode--font-lock-defaults
+  `(
+    ;; Builtins for BUILD files are a subset of Starlark keywords. For details
+    ;; see https://docs.bazel.build/versions/master/skylark/language.html.
+    (,(regexp-opt '("exports_files" "glob" "licenses" "package"
+                    "package_group" "select" "workspace")
+                  'symbols)
+     . font-lock-builtin-face)
```

Comment:
I think the face symbols need to be quoted. This only works because there's a deprecated variable `font-lock-builtin-face` with value `font-lock-builtin-face` etc.

Reply:
Done. Thanks for pointing this out!